elephant_vending_machine/views.py

```diff
@@ -48,17 +52,36 @@ def upload_image():
     will be returned
 
     Returns:
-        HTTP response 200 with payload 'File received' or
-        HTTP response 500 with payload 'No image file in request.'
-
+        HTTP response 200 with payload 'Success: Image saved.' or
```

Comment:
Might want to consider a format similar to [this post](http://kartowicz.com/dryobates/2016-10/sphinx-rest-api/) for API endpoint docs. It'd be a bit verbose though for a docstring... Mainly I like the idea of having example requests and responses

Reply:
See latest commit for this, I included sample request and response with the nice formatting for sphinx. For whatever reason POSTs don't look as nice, but I think this is a sphinx issue.